modules/services/window-managers/i3-sway/lib/functions.nix

```diff
@@ -85,7 +85,7 @@ rec {
               (optionalString (position != null) "position ${position}")
               (optionalString (statusCommand != null)
                 "status_command ${statusCommand}")
-              "${moduleName}bar_command ${command}"
+                "${moduleName}bar_command ${command}"
```

Comment:
Have you actually tested this? It looks like these spaces are in a Nix expression context, so they will just be ignored.

Do you actually need this for the configuration to be valid or is it just cosmetic?